pkg/devicemapper/devmapper.go

```diff
@@ -264,11 +264,6 @@ func UdevWait(cookie *uint) error {
 	return nil
 }
 
-// LogInitVerbose is an interface to initialize the verbose logger for the device mapper library.
-func LogInitVerbose(level int) {
```

Comment:
Ok, given other core interfaces like `RemoveDevice` and `DeleteDevice` rely on `dmSawBusy` and it will be set only if user of package calls `LogInit()` (Either with or without a logger). Otherwise package will always return error even in case of busy device and send messages from libdm to stdout.

IOW, for devicemapper pkg to work properly, this package always need to register a callback function with libdm. And right now unfortunately it is initiated from the user of package.

So I agree that cleaning this up makes sense. While there might be a very strange user out there but given how twisted this internface is, chances of this happening are small. If it does happen, we will try to advise them to move to new interface.

So, I am fine with removing `LogInitVerbose`. Please do capture some of this information in changelog for future reference.